lib/internal/buffer_old.js

```diff
@@ -11,6 +11,7 @@ const kMaxLength = smalloc.kMaxLength;
 exports.Buffer = Buffer;
 exports.SlowBuffer = SlowBuffer;
 exports.INSPECT_MAX_BYTES = 50;
+exports.kMaxLength = binding.kMaxLength;
```

Comment:
exposing `kMaxLength` does keep parody with how it's exposed today via smalloc. and IIRC it was also exposed before smalloc.

That value is more useful for users now because the maximum size of allocation depends on the architecture. So serves as a preemptive means to prevent an exception.

In reality I don't care that much, but it would make testing more difficult.